File: app/src/main/java/fr/openium/auvergnewebcams/custom/SimpleImageLoaderCallback.java
package fr.openium.auvergnewebcams.custom;

import com.github.piasy.biv.loader.ImageLoader;

import java.io.File;

public class SimpleImageLoaderCallback implements ImageLoader.Callback {

    @Override
    public void onCacheHit(int imageType, File image) {

    }

    @Override
    public void onCacheMiss(int imageType, File image) {

    }

    @Override
    public void onStart() {

    }

    @Override
    public void onProgress(int progress) {

    }

    @Override
    public void onFinish() {

    }

    @Override
    public void onSuccess(File image) {

    }

    @Override
    public void onFail(Exception error) {

    }
}

File: app/src/main/java/fr/openium/auvergnewebcams/custom/SimpleImageLoaderCallback.kt
package fr.openium.auvergnewebcams.custom

import com.github.piasy.biv.loader.ImageLoader
import java.io.File

open class SimpleImageLoaderCallback : ImageLoader.Callback {
    override fun onCacheHit(imageType: Int, image: File) {}
    override fun onCacheMiss(imageType: Int, image: File) {}
    override fun onStart() {}
    override fun onProgress(progress: Int) {}
    override fun onFinish() {}
    override fun onSuccess(image: File) {}
    override fun onFail(error: Exception) {}
}